src/problem-analysis/state-of-the-art/chess-engines.tex

```diff
@@ -1,4 +1,108 @@
 % textidote: ignore begin
 \subsection{Chess engines}\label{subsec:chess.engines}
+
+Chess engines are defined by \href{https://www.chess.com}{\verb{Chess.com}} as such:
```

Comment:
@SebastianMygind I found the error: `\verb` is invalid here

Reply:
Nice it compiles now